lib/brewdler/brew_installer.rb

```diff
@@ -2,14 +2,14 @@ module Brewdler
   class BrewInstaller
     def self.install(name, options = {})
       if `which brew`; $?.success?
-        command = "brew install #{name}"
+        command = [name]
         unless options[:args].nil?
           options[:args].each do |arg|
-            command << " --#{arg}"
+            command << "--#{arg}"
           end
         end
 
-        `#{command}`
+        system "brew", "install", *command
```

Comment:
I think it should start with a `--verbose` flag/environment variable for now and we could talk about changing the behaviour in a future PR.